CHANGELOG.md

```diff
@@ -2,8 +2,23 @@
 
 All notable changes to this project will be documented in this file.
 
-## [v8.6.1](https://github.com/nextcloud-libraries/nextcloud-vue/tree/v8.6.1) (2024-02-01)
```

Comment:
Have you lost a previous release links intentionally?)

Reply:
Oooops. Nice catch 💙

Reply:
Fixed 🙈

Reply:
And Full changelog from 8.6.1?